lib/util/SparqlExpressionEvaluator.js

```diff
@@ -202,6 +204,9 @@ operators = {
   operators[operatorName].resultType = 'boolean';
 });
 
+// Tag all operators with optional arguments
+operators.regex.optionalArguments = 1;
```

Comment:
Is this the number of optional arguments? In that case, `optionalArgLength` might be more intuitive.